### DISCUSSION TITLE
Using AzuraCast in Secondlife  -Shoutcast board not showing information

### DISCUSSION THREAD

**Discussion by ChaoticVortex** [AzuraCast/AzuraCast#6896] [Q&A]
First off if this post is in the wrong section, my apologies I am not the sharpest tool in the shed when it comes to coding, and if was not for Buster Neece's showing step by step how to get started with Linode, I would not be here right now writing this. I am not even sure if this is a long shot or not, but I figured it could not hurt to ask.

A quick TL;DR version up to this point is that I have been DJing on Second Life for some time now, so I know how to operate a shout Cast board. and all that good stuff. Also, the stream on the public pages does what it should, however I have this weird issue and I am not sure how to cure it.

I put the stream url of http:/ip address given by azuracast:8000/live and the song information comes up with zero issue in local chat, however when I add that to the shoutcast board, I get the message. "We couldn't reach this radio station, or its stream type is not supported". At first, I thought it was because I was not running Shoutcast itself, so I followed the instructions to add it, but I am getting the same message.  So I thought it must be the API's so I added: /nowplaying, /nowplaying/{station_id}, and /station/{station_id}/streamers, still getting the same message

Is there something else I need to do through linode? Or is there something else I have to do?

**Comment by BusterNeece** [comment]
@ChaoticVortex You actually can use Shoutcast if you need to by visiting the "Install Shoutcast" page on system administration, downloading and installing the latest Shoutcast binary from their web site, and then switching to it in your station's profile.

While we can't ship Shoutcast directly because their software is proprietary and not open-source, we do support using it if need be, and if you're using something that depends on Shoutcast's specific API, that might be one such case. We wouldn't recommend using it otherwise, though, as Icecast has specific improvements that we take advantage of (and supports more bitrates for free).

Our Now Playing API is indeed quite useful (and has a _lot_ more information than either the Icecast or Shoutcast APIs could provide), but most software that's looking for Icecast or Shoutcast won't understand it natively.

**Reply by ChaoticVortex** [reply]
Thank you for the reply. Like I said everything is running smoothly, I appreciate the time you put into making the video so people like me who do not knot the first thing about this stuff and get up and running in moments.

Though it is not technically important to running the radio station, and though Second Life is its own beast, I am just trying to figure out where I maybe went wrong for sending the information to DJ boards in Second life :D
